src/pages/Home.js

```diff
@@ -22,6 +22,7 @@ import Lists from '../utils/Lists'
 import useAuth from '../hooks/useAuth'
 import Context from '../modules/Context'
 import userStory from '../services/user_story'
+import productService from '../services/product'
```

Comment:
This gives an idea that we are maintaining product as a separate service in the backend although in backend we have only 1 service/api. Lets keep these in the `services/user_story` file itself.